src/app/components/Metadata/index.test.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Helmet } from 'react-helmet';
```

Comment:
I'm curious to know, do we need to use react helmet here, or are we using it because it is better? (I noticied we're using Helmet.peek() which isn't something I'd seen before!)

Reply:
Great question, and thank you for asking! 

So, when I was tidying up the tests, I noticed that the snapshot tests were all for empty divs e.g. https://github.com/bbc/simorgh/blob/44461633623118ce2617962a91f44167907ffb89/src/app/legacy/containers/Metadata/__snapshots__/index.test.jsx.snap. This was not what I was expecting, so I did a little bit of Googling 😊  and I found that rendering using Helmet doesn't actually return any content - which is why all of the snapshots were empty. I found an example using `Helmet.peek()` with seems to return the entire Helmet object, and I was able to snapshot that instead. Hope that helps! 

I also used the same thing in the LinkedData PR - in that example, I wrote a function to extract the first script element out of the Helmet - https://github.com/bbc/simorgh/blob/latest/src/app/components/LinkedData/index.test.tsx#L16-L21, and snapshotted that - you'll see proper snapshots, and not lot of empty divs! https://github.com/bbc/simorgh/blob/latest/src/app/components/LinkedData/__snapshots__/index.test.tsx.snap

Hope this helps, and let me know if you have any other questions!